ptero_lsf/implementation/backend.py

```diff
@@ -219,6 +219,12 @@ def update_job_status(self, job_id):
 
                 LOG.exception("Exception occured while converting lsf"
                     " job to dictionary")
+
+                if (service_job.failed_update_count >= MAX_FAILS):
```

Comment:
I'm surprised that MAX_FAILS was already in this module... it was unused before this.